R/IonDb.R

```diff
@@ -0,0 +1,224 @@
+#'
+#' @section Retrieve annotations and ion information from the database:
+#'
+#' Annotations/compound informations can be retrieved from a `IonDb` in the same
+#' way as thay are extracted from a `CompDb`. In addition to that the function 
+#' `ions` allows to retrieve ion information from the `IonDb` object. It returns 
+#' the actual data as a `data.frame` (if `return.type = "data.frame"`) or a 
+#' [tibble::tibble()] (if `return.type = "tibble"`). A `ions` call will always 
+#' return all elements from the *ms_ion* table (unless a `filter` is used).
+#'
+#' @section General functions (beside those inherited from `CompDb`):
+#'
+#' - `IonDb`: connect to a compound/ion database.
+#'
+#' - `ionVariables`: returns all available columns/database fields for ions.
+#' 
+#' - `insertIon`: allows to add further ions to the `IonDb` object.
+#'
+#'
+#' @section Filtering the database:
+#'
+#' Like `compounds` and `Spectra` also `ions` allows to filter the
+#' results using specific filter classes and expressions. Filtering uses the
+#' concepts from Bioconductor's `AnnotationFilter` package. All information
+#' for a certain compound with the ID `"1"` can for example be
+#' retrieved by passing the filter expression `filter = ~ ion_id == "1"` to 
+#' the `ions` function.
+#'
+#' Use the [supportedFilters] function on the [IonDb] object to get a list of
+#' all supported filters. See also examples below or the usage vignette for
+#' details.
+#'
+#' @param columns For `ions`: `character` with the names of the database 
+#'     columns that should be retrieved. Use `ionVariables` for a list 
+#'     of available column names.
+#'
+#' @param filter For `ions`: filter expression or [AnnotationFilter()] defining 
+#'     a filter to be used to retrieve specific elements from the database.
+#'
+#' @param includeId for `ionVariables`: `logical(1)` whether the ion
+#'     ID (column `"ms_ion_id"`) should be included in the result. The
+#'     default is `includeId = FALSE`.
+#'
+#' @param object For all methods: a `IonDb` object.
+#'
+#' @param return.type For `ions`: either `"data.frame"` or `"tibble"` to
+#'     return the result as a [data.frame()] or [tibble()], respectively.
+#'
+#' @param x For `IonDb`: database connection (should be to empty database?) or 
+#'     `character(1)` with the file name of the SQLite database where the 
+#'     `IonDb` data will be stored.
+#'
+#'     For all other methods: a `IonDb` object.
+#' 
+#' @param cdb for `IonDb`: `CompDb` object used to construct a `IonDb` object 
+#'     (the content of `cdb` is copied to the `IonDb` object).
+#' 
+#' @param ions for `IonDb`: table to be added to the `IonDb` object (if missing
+#'     an empty table is added). `ions` is required to have the columns 
+#'     "compound_id", "ion_adduct", "ion_mz" and "ion_rt". 
+#'
+#' @param ... additional arguments. Currently not used.
+#'
+#' @author Andrea Vicini, Johannes Rainer
+#'
+#' @examples
+#'
+# # I have to improve/complete this
```

Comment:
Looks like here you miss all the `'` in the `#'`.

Reply:
You are right, I'll fix this